extensions/wikia/UserRenameTool/UserRenameToolProcess.class.php

```diff
@@ -555,11 +673,17 @@ private function doRun() {
 				$fakeUser->addToDatabase();
 			}
 
-			$fakeUser->setGlobalAttribute( 'renameData', self::RENAME_TAG . '=' . $this->mNewUsername . ';' . self::PROCESS_TAG . '=' . '1' );
+			$fakeUser->setGlobalAttribute(
+				'renameData',
+				self::RENAME_TAG . '=' . $this->mNewUsername . ';' . self::PROCESS_TAG . '=' . '1'
```

Comment:
Given all the manual serialization I see in this code, it might be worthwhile to change all this to JSON formatted strings and use `json_encode` and `json_decode` as part of the cleanup.